model-optimizer/mo/utils/ir_reader/layer_to_class.py

```diff
@@ -233,6 +233,21 @@ def assign_add_output_result(op: Node):
     op.out_port(0).connect(tmp_result.in_port(0))
 
 
+def topk_add_output_result(op: Node):
```

Comment:
The code of this function is identical to the code in the TopKNormalizer. Let's update the TopKNormalizer transformation and this function to it. Also update the IR Reader to use the function from new location.

Reply:
Normalizer and reader function updated.